packages/hmr-plugin/src/hmr-bootstrap.ts

```diff
@@ -15,9 +16,6 @@ export async function hmr<T>(
 
   webpackModule.hot.accept();
 
-  interface HmrDataTransfer {
-    snapshot?: any;
-  }
```

Comment:
This is only used within this method as far as I can see, so we should not move it to the symbols file.
It should stay here as internal implementation detail.
Unless I am missing something...

Reply:
done